.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: go
 
 go:
-  - "1.5"
-  - "1.10"
+  - "1.15"
```

Comment:
I think you'll need to edit the `script` section below to get the `go test` command to test the root module and the v2 module